### ISSUE TITLE
(One#26774793) [Add an action - Search]: Screen reader doesn't announce the search results information by entering any valid/Invalid data in the "Search" edit field.

### ISSUE THREAD

**Issue by ek68794998** [Azure/LogicAppsUX#4296]
### **User Experience:** 

Screen reader dependent users will not be able to know whether their entered input has shown some results or not if the screen reader doesn't announce the search results information by entering any valid/invalid data in the search edit field.

### **Pre-requisite:** 

- Turn on the Screen reader (Narrator). 

### **Repro Steps:** 

1. Open URL "https://make.powerautomate.com/" and sign in with Valid credentials. 
2. Flow Homepage will appear on the screen, navigate to 'My Flows' present on left nav and select it. 
3. For a flow already created, click on 'Edit'. 
4. Edit page will appear, navigate to 'New Step' button and select it. 
5. Choose an Action section will appear on the screen, Navigate to 'Search Connectors & Actions' edit field and search for valid/invalid input. 
6. Verify whether screen reader is announcing the search results for valid/invalid input 

### **Actual Result:** 

Screen reader doesn't announce the search results information by entering any valid/Invalid data in the "Search" edit field.

**Observation:** By entering any Valid/Invalid data, the screen reader does not announce the search results information, instead it is remaining silent.

Other screen reader like JAWS and NVDA show same behaviors as they do not announce the search results information.

### **Expected Result:** 

Screen reader should announce the search results information by entering any valid/Invalid data in the "Search" edit field.

**Example:** By entering any Valid data (Ex: When an email arrives) the screen reader should announce as "One result found" and by entering any invalid data (Ex: rhhjfbjbefi) the screen reader should announce as "No results found, or Zero results found".

### **MAS Reference:** 

[MAS 1.3.1 -- Info and Relationships](<https://aka.ms/MAS1.3.1>)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**

NA

**UIA/DOM details available:**

Yes

### **Reference Links:** 

- [Accessibility Insights!](<https://accessibilityinsights.io/>) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![MAS1 3 1 Screen reader doesn't announce the search results information by entering any valid or Invalid data in the Search edit field](<https://github.com/Azure/LogicAppsUX/assets/1350074/9ffb6d9d-115e-41d6-8fc3-f4f193d480c8>)

### Browser 

- **OS Version:** Windows 11 Version 23H2 (OS Build 22631.3007) 
- **Browser:** Edge Version 120.0.2210.144 (Official build) (64-bit) 
- **\#URL:**[https://make.powerautomate.com/](<https://make.powerautomate.com/>) 
- **Screen Reader:** Narrator 

### Additional context 

[One#26774793](<https://dev.azure.com/msazure/One/_workitems/edit/26774793>)

**Comment by hartra344** [comment]
Fixed here: #4082
